lib/utils.js

```diff
@@ -170,3 +170,18 @@ function basename(path) {
   return (path === '..' || path === '.' ? '' : path);
 }
 exports.basename = basename;
+
+
+function getLimit(limits, name, defaultLimit) {
+  if (!limits) return defaultLimit;
+
+  var limit = limits[name];
+  if (limit === undefined) return defaultLimit;
```

Comment:
I think this should probably be `==` instead, with a comment above mentioning that the `==` is intentional (to match both `undefined` and `null`).